PurchasesCoreSwiftTests/Purchasing/StoreKitWrapperTests.swift

```diff
@@ -146,7 +148,7 @@ class StoreKitWrapperTests: XCTestCase, RCStoreKitWrapperDelegate {
 
         wrapper?.paymentQueue(paymentQueue, updatedTransactions: [transaction])
 
-        wrapper?.finish(transaction)
+        wrapper?.finishTransaction(transaction)
```

Comment:
this is a slight naming change, but it more closely resembles the swift interface for [this method](https://developer.apple.com/documentation/storekit/skpaymentqueue)